gravatar/src/main/java/com/gravatar/di/container/GravatarSdkContainer.kt

```diff
@@ -10,9 +10,15 @@ import okhttp3.OkHttpClient
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
-internal class GravatarSdkContainer private constructor() {
-    companion object {
-        val instance: GravatarSdkContainer by lazy {
+/**
+ * Container that handle the DI for the Gravatar SDK.
+ */
+public class GravatarSdkContainer private constructor() {
```

Comment:
My main concert is about making this class public to be used in the `gravatar-ui ` module. I wanted to use the dispatcher in the [GravatarImagePickerWrapper.kt](https://github.com/Automattic/Gravatar-SDK-Android/pull/151/files#diff-30259c0c1479d08a6f6951482b4f25dee6b030d4e33db9ea7ccc01ffefbbe529), but maybe I should look for another option. Wdty?